content/source/docs/cloud/api/run.html.md

```diff
@@ -103,6 +106,9 @@ Status  | Response                               | Reason
     "attributes": {
       "is-destroy": false,
       "message": "Custom message",
+      "refresh": false,
+      "refresh-only": false,
+      "replace-addrs": ["example.resource_address"],
```

Comment:
Might be good to leave the sample payload as a _typical_ example, not just one that's exhaustive of all options. Specifying all options with their defaults is overly verbose and creates weird things like seeing `target` and `replace` stuff together, etc; under this logic I'd even just remove the existing `is-destroy`.  Thoughts?

Reply:
I think I second this.

Reply:
Ah, good point!